### DISCUSSION TITLE
Layered images on top of a TPanel

### DISCUSSION THREAD

**Discussion by chicknsoup** [AngusJohnson/Image32#40] [General]
Hi, 
Can I create layered images on top of a TPanel?
An example would be nice.

Thank you for the awesome library.

**Comment by AngusJohnson** [comment]
I've attached a quick sample application in the zip file below.
However, unless you have good reason to use TPanels as the image container I'd recommend using TPaintBox controls instead for 2 reasons:
1. They are transparent controls
2. They publish an OnPaint event which makes them much easier to use when custom painting.

![image32_sample](https://github.com/AngusJohnson/Image32/assets/5280692/ad6a6a79-6758-4845-b7f7-410e53336935)
[image32_sample.zip](https://github.com/AngusJohnson/Image32/files/13644330/image32_sample.zip)

**Reply by chicknsoup** [reply]
Thank you so much.
